### PR TITLE
feat: Lifi

### PR DESCRIPTION
**Pull request by skeremidchiev**
This PR includes:
1) `LiFi` swap provider integration in `wallet-core` package
2) `LiFi` error handling in `error-parser` package
3) Changes in `error-parser`.

### CODE REVIEW

File: packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+  }
+
+  wrapUserErrorMessage(context: HighInputAmountErrorContext, lang?: string): void {
+    const { expectedMaximum, assetCode } = context;
+    switch (lang) {
+      default:
+        this.userMsg = {
+          cause: `Input amount is too high, please decrease it`,
+          suggestions: [],
```
Comment (koderholic) [inline packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts]: @skeremidchiev Could you express cause as a statement and suggestions as an array of actions the user can possibly take? i.e 
`cause: `Input amount is too high, expected maximum amount is ${expectedMaximum} ${assetCode}`, suggestions: ['Please decrease it']`

File: packages/error-parser/src/LiqualityErrors/InsufficientInputAmountError.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const { expectedMinimum, assetCode } = context;
+    switch (lang) {
+      default:
+        this.userMsg = {
+          cause: `Input amount is too low, please increase it`,
+          suggestions: [],
+        };
+        if (expectedMinimum && assetCode) {
+          this.userMsg.suggestions.push(`Expected minimum amount is ${expectedMinimum} ${assetCode}`);
+        }
```
Comment (koderholic) [inline packages/error-parser/src/LiqualityErrors/InsufficientInputAmountError.ts]: Same as above

**Review by koderholic — CHANGES_REQUESTED** [review]
Hey @skeremidchiev Great so far on error-parser, just a few comments from me.

File: packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts (hunk 1 above)
Comment (skeremidchiev) [reply packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts]: Done.

File: packages/error-parser/src/LiqualityErrors/InsufficientInputAmountError.ts (hunk 2 above)
Comment (skeremidchiev) [reply packages/error-parser/src/LiqualityErrors/InsufficientInputAmountError.ts]: Done.

File: packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts (hunk 1 above)
Comment (koderholic) [reply packages/error-parser/src/LiqualityErrors/HighInputAmountError.ts]: Hey @skeremidchiev Didn't really see any changes

**Review by koderholic — APPROVED** [review]
Please update LiFi in wallet-core to use the parser. Else, looking good for error-parser

**Pull request merged**